.env

```
@@ -1,2 +1,8 @@
 PUBLIC_SUPABASE_URL=http://127.0.0.1:64321
 PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0
+PUBLIC_ORIGIN='http://localhost:5174'
```

Let's use the default port or we can change the port manually for local testing to e.g. 5200
```suggestion
PUBLIC_ORIGIN='http://localhost:5173'
```